phpstan.neon

```diff
@@ -5,12 +5,7 @@ includes:
     - vendor/phpstan/phpstan-beberlei-assert/extension.neon
     - vendor/phpstan/phpstan-deprecation-rules/rules.neon
 
-
 rules:
-    - Sourceability\CodingStandard\PHPStan\Rules\ConstantBooleanIdenticalRule
```

Comment:
Did you run into issues with phpstan? I would like to understand why those rules were removed?

Reply:
<img width="1139" alt="Screen Shot 2021-02-10 at 9 57 43 AM" src="https://user-images.githubusercontent.com/1288852/107550895-7228d380-6b86-11eb-80b6-72ad18387d8b.png">

As far as I can tell, those rules do not exist.

Reply:
Maybe something isn't importing correctly?

https://github.com/sourceability/phpstan-rules/tree/master/src/Rules

Reply:
That dependency `sourceability/phpstan-rules` is currently not required by this project. I will add it then.

Reply:
AH! I probably did the same thing 😆 I linked to the proper repo `phpstan-rules` but saw `"sourceability/coding-standard": "^1.0@dev",1 in `composer.json` and just glossed over it as "The standard are there"